sierra_adapter/sierra_items_to_dynamo/src/main/scala/uk/ac/wellcome/platform/sierra_items_to_dynamo/dynamo/package.scala

```diff
@@ -0,0 +1,34 @@
+package uk.ac.wellcome.platform.sierra_items_to_dynamo.dynamo
+
+import com.gu.scanamo.DynamoFormat
+import com.gu.scanamo.syntax._
+import uk.ac.wellcome.models.transformable.sierra.{
+  SierraItemRecord,
+  SierraRecordNumber
+}
+import uk.ac.wellcome.storage.dynamo._
+import uk.ac.wellcome.storage.type_classes.IdGetter
+
+object dynamo {
+  implicit val recordNumberFormat =
+    DynamoFormat
+      .coercedXmap[SierraRecordNumber, String, IllegalArgumentException](
```

Comment:
Actually same goes for this..Id' rather have those in their companion object than in a package that then needs to be imported every time they're needed